spec/terraform_template_renderer_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'tmpdir'
+
+require 'TerraformDevKit/config'
+require 'TerraformDevKit/environment'
+require 'TerraformDevKit/terraform_template_renderer'
+
+TDK = TerraformDevKit
+
+RSpec.describe TDK::TerraformTemplateRenderer do
+  let(:tf_template) do

```

Comment:
Can we add here a `describe` and `context` block  to wrap all this set up to make it a little more clear which method is being tested and under which context. 

Would it also be good to add another context to test the default path when no extra template files are passed?